config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  match '/*all', to: 'application#cors_preflight_check', via: [:options]
```

Comment:
What does this route match? Aren't routes regex? (In which case this would match `/////all` and such.)